pallets/creditcoin/src/types.rs

```diff
@@ -53,6 +53,16 @@ pub enum Blockchain {
 	Other(OtherChain),
 }
 
+#[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub struct BurnId(pub u64);
+
+#[derive(Clone, Encode, Decode, Eq, PartialEq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+pub struct BurnInfo<AccountId, Balance> {
+	pub account: AccountId,
+	pub balance: Balance,
```

Comment:
Can we call it `amount` instead of `balance`? I feel like talking about the `amount` of a burn is more natural than `balance`.